### ISSUE TITLE
We need a FAQ that explains roles better

### ISSUE THREAD

**Issue by mikeysan** [EddieHubCommunity/support#259]
Could we please have a description of roles that are assignable to users on discord with an explanation of when they are assigned?

**Comment by UnevenCoder** [comment]
Can I start adding ?

**Comment by eddiejaoude** [comment]
Thanks @mikeysan .

Yes @UnevenCoder , please start adding, we dont need everything at once. Even 1 or 2 roles with description add value to the community 👍

**Comment by BOLT04** [comment]
@mikeysan this is indeed necessary, thank you for raising the issue 👍. Do you think this FAQ can simply be in this repository, or could it also be on `EddieBot`?

**Comment by eddiejaoude** [comment]
I think we need a home for general community FAQs, where EddieBot can read it from there. Also each repo might have its own FAQs.

**Comment by UnevenCoder** [comment]
I think eddie bot will be better

**Comment by eddiejaoude** [comment]
We can always move it later, it would be best to document it where it is the most convenient for now. It will add value once it is available which more important than finding the "perfect" location, which wont always be perfect as our project/community evolve  🤓 

In short @UnevenCoder I think `support` would be best and you think `EddieBot` - I think both are right, so whoever does this can decide for now 😄

**Comment by UnevenCoder** [comment]
cool why not both ?

**Comment by BOLT04** [comment]
@UnevenCoder If the info is duplicated in both places, when we change on one place, we need to remember to change on the other place as well 😅. For that reason, I think we should have it on the `support` repo which is more "abstract" or higher level, then on `EddieBot` we add a link to the `support` FAQ. So we can make a change to both places, but with the role explanation on only one.

**Comment by eddiejaoude** [comment]
I agree with @BOLT04 - we want it in a single place and `support` is best, then we can always easily link to it, rather than having docs mixed in with functionality.

**Comment by eddiejaoude** [comment]
Stalebot we still need this 🤓

**Comment by BOLT04** [comment]
I think we still need this stalebot

**Comment by eddiejaoude** [comment]
Maybe this could be added to the `#README` channel, this is the supporting doc https://github.com/EddieHubCommunity/support/blob/main/discord-readme.md

**Comment by mikeysan** [comment]
Still needed
